### PR TITLE
docs: Update Orchestrator requirement docs in prep for Composer 1.3.1

### PR DESCRIPTION
**Pull request by taicchoumsft**
## Description
Add documentation for Orchestrator preview. Also make clear that it's possible to publish bot again in 32-bit, even though Composer itself has to run in 64-bit.

## Task Item
#minor

### CODE REVIEW

File: docs/preview features/orchestrator.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Orchestrator Preview
+Orchestrator is a powerful new language model that is optimized for conversational AI applications.
```
Comment (tsuwandy) [inline docs/preview features/orchestrator.md:2]: "Orchestrator is a powerful LU solution optimized for conversational AI applications.".

File: docs/preview features/orchestrator.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+- [Technical Overview](https://github.com/microsoft/botframework-sdk/blob/main/Orchestrator/docs/Overview.md)
+- [Samples](https://github.com/microsoft/BotBuilder-Samples/tree/main/experimental/orchestrator)
+
+## Orchestrator Features
+- Easy to use - Use your existing LU files from LUIS without modification
+- State of the Art - Extensive pre-trained transformer-based model for conversational applications
+- Extremely fast Inference Time: Written in C++ for performance
```
Comment (tsuwandy) [inline docs/preview features/orchestrator.md:10]: inference time

File: docs/preview features/orchestrator.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
+# Orchestrator Preview
+Orchestrator is a powerful new language model that is optimized for conversational AI applications.
+
+- [Technical Overview](https://github.com/microsoft/botframework-sdk/blob/main/Orchestrator/docs/Overview.md)
+- [Samples](https://github.com/microsoft/BotBuilder-Samples/tree/main/experimental/orchestrator)
+
+## Orchestrator Features
+- Easy to use - Use your existing LU files from LUIS without modification
+- State of the Art - Extensive pre-trained transformer-based model for conversational applications
```
Comment (tsuwandy) [inline docs/preview features/orchestrator.md:9]: State of the art - Extensive pre-trained transformer-based models for conversational applications

File: docs/preview features/orchestrator.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+- Extremely fast Inference Time: Written in C++ for performance
+- Cross-platform Support: Runs on Windows, Linux and Mac OS X, including Cloud platforms
```
Comment (tsuwandy) [inline docs/preview features/orchestrator.md:11]: Cross-platform support...

File: docs/preview features/orchestrator.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+- Free to use: Language models are deployed alongside your bot, there are no usage fees
```
Comment (tsuwandy) [inline docs/preview features/orchestrator.md:12]: Also runs in memory.  No calling a service (ie LUIS)

**Review by tsuwandy — APPROVED** [review]

File: docs/preview features/orchestrator.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+## Getting Started
+------------------
+### Limitations
+For this preview release, please be aware of these limitations:
+
+1. Orchestrator has to be deployed on an x64 platform. If you are deploying your bot to a cloud service, please be aware that you need to publish your bot with `win-x64` enabled, and enable x64.
+
+1. We are supporting English-only for this release.
+
+1. Orchestrator does not support Entity extraction for this release. Any Entities declared in your LU file will be ignored.
```
Comment (zxyanliu) [inline docs/preview features/orchestrator.md:23]: lowercase "Entity"/ "Entities".

File: docs/preview features/orchestrator.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+Please see the [roadmap](https://github.com/microsoft/botframework-sdk/blob/main/Orchestrator/docs/Overview.md#Roadmap) for when these features will be added to Orchestrator.
+
+
+### Adding Orchestrator to a new bot in Composer
+1. Enable the preview feature in Orchestrator:
```
Comment (zxyanliu) [inline docs/preview features/orchestrator.md:29]: "Enable Orchestrator from the preview features list:"

File: docs/preview features/orchestrator.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+    - Navigate to `Application Settings` (
+    <img src="orchestrator-assets/application-settings.png" alt="drawing" width="30"/> icon)
+    - Enable Orchestrator preview feature under `Applicatio Updates -> Preview features`
+
+      ![orchestrator flight](orchestrator-assets/preview-flag-orchestrator.png)
+
+1. Create a new bot, or open an existing bot project. Select a dialog in the bot project that you would like to try Orchestrator with. Select `Orchestrator recognizer` from the `Recognizer Type` dropdown
```
Comment (zxyanliu) [inline docs/preview features/orchestrator.md:36]: "Create a new bot or open an existing bot project."
"Select `Orchestrator recognizer` from the `Recognizer Type` drop-down menu."

File: docs/preview features/orchestrator.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+
+   <img src="orchestrator-assets/orchestrator-recognizer-dropdown.png" alt="recognizer dropdown options" width="300">
+
+1. Create an intent trigger
+    - Click on the waffle menu on a dialog to add a new trigger
+    ![add a trigger](orchestrator-assets/add-trigger.png)
+    - Create a new `Intent recognized` trigger, and fill in the intents for your dialog in LU format. For more details on the LU format, please see [here](https://docs.microsoft.com/en-us/azure/bot-service/file-format/bot-builder-lu-file-format?view=azure-bot-service-4.0)
```
Comment (zxyanliu) [inline docs/preview features/orchestrator.md:43]: please see [here](https://docs.microsoft.com/azure/bot-service/file-format/bot-builder-lu-file-format).

File: docs/preview features/orchestrator.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+    - Add a response to the new `Intent Recognized` trigger to test that Orchestrator is working
+        - In the flow diagram for the newly created trigger, click on the `+` button and select `Send a Response` from the dropdown, then add a sample response in the `Language Generation` textbox.
+
+1. Modify the runtime to support Orchestrator
+    1. Eject the runtime to add Orchestrator libraries to the runtime
+        1. Navigate to `Application Settings` (
+            <img src="orchestrator-assets/application-settings.png" alt="drawing" width="30"/>) and toggle the `Use custom runtime` switch. Then click on `Get a new copy of the runtime code` and select `C#` to eject a copy of the dotnet runtime for your bot. Click `Okay`.
+
+            ![eject-runtime](orchestrator-assets/eject-runtime.png)
+        1. The runtime will be ejected into the `<Bot Project Folder>/runtime/azurewebapp` folder of your  bot project.
```
Comment (zxyanliu) [inline docs/preview features/orchestrator.md:53]: "The runtime will be ejected into the `<Bot Project Folder>/runtime/azurewebapp` folder of your bot project."
An extra space.

File: docs/preview features/orchestrator.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+            - Add `Microsoft.Bot.Builder.AI.Orchestrator` after `Microsoft.Bot.Builder.AI.Luis`
+                ```xml
+                <ItemGroup>
+                <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.2" />
+                <PackageReference Include="Microsoft.Bot.Builder" Version="4.11.0" />
+                <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.11.0" />
+                <PackageReference Include="Microsoft.Bot.Builder.AI.Orchestrator" Version="4.11.0-preview" />
+                ...
+                ```
+                **Important: All packages starting with `Microsoft.Bot.Builder` must have the same version numbers. For example, if your ejected runtime `Microsoft.Bot.Builder` version is `4.11.0` like above, set the `Microsoft.Bot.Builder.AI.Orchestrator` version to `4.11.0-preview`.**
```
Comment (zxyanliu) [inline docs/preview features/orchestrator.md:67]: You may consider using a text box for the important note. 
Here is the syntax: 

> [!Important]
> All packages starting with `Microsoft.Bot.Builder` must have the same version numbers. For example, if your ejected runtime `Microsoft.Bot.Builder` version is `4.11.0` like above, set the `Microsoft.Bot.Builder.AI.Orchestrator` version to `4.11.0-preview`.

File: docs/preview features/orchestrator.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+
+   Orchestrator has to run on a 64-bit platform. A `BadImageFormatException` indicates that a 32-bit dotnet SDK was used to compile the bot, or that the bot was run on a 32-bit or unsupported platform.
+
+   Please *uninstall* any 32-bit dotnet SDKs that are on your local machine. Download the latest [v3.1 x64 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1) for your platform.
+
+   Finally, be sure that you're deploying and running your bot on a 64-bit capable machine.  If you've deployed your bot via Composer to Azure, this is automatically taken care of for you.
+
+1. How do I run Orchestrator on a NodeJS-based bot?
+
+   Please see the instructions [here](https://aka.ms/bf-orchestrator#in-a-javascript-bot).  Then eject a `NodeJS` runtime instead of a `C#` one.
```
Comment (zxyanliu) [inline docs/preview features/orchestrator.md:120]: "Node.js"

**Comment by zxyanliu** [comment]
@taicchoumsft thanks for providing the content.
I did a rough language and formatting pass and you can see them in my comments. 

Here are a few additional comments:
- Should we update the **Learn more** link of Orchestrator in Composer UI and point users to the `orchestrator.md` file you wrote with detailed instructions for enabling Orchestrator in Composer? 
- Can we change all headings to sentence case instead of title case? 
- Can we add grey borders for these images you included in the `orchestrator.md` file so that they have better visibility among the texts?

**Review by zxyanliu — DISMISSED** [review]

File: docs/preview features/orchestrator.md (hunk 5 above)
Comment (taicchoumsft) [reply docs/preview features/orchestrator.md:12]: Thanks for your input @tsuwandy, all comments addressed.

File: docs/preview features/orchestrator.md (hunk 12 above)
Comment (taicchoumsft) [reply docs/preview features/orchestrator.md:120]: Thanks @zxyanliu , all changes made.

**Comment by taicchoumsft** [comment]
> @taicchoumsft thanks for providing the content.
> I did a language and formatting pass and provided some minor suggestions.
> 
> Here are a few additional comments:
> 
> * Should we add a **Learn more** link of Orchestrator in Composer UI and point users to the `orchestrator.md` file? If so, you will need an aka link for this doc.
> * Can you change all headings to sentence case instead of title case?

Yes, we currently have a aka link pointing to another Orchestrator doc (aka.ms/bf-orchestrator), I'll sync with our PM to make the change if necessary. 

Made sentence case change.

**Review by scheyal — APPROVED** [review]
looks good. thank you.

**Review by boydc2014 — APPROVED** [review]

**Review by boydc2014 — APPROVED** [review]

**Pull request merged**